README.md

```diff
@@ -52,7 +52,8 @@ We will start from the lowest-level definitions, and work upwards.
 
 Build the go code with:
 ```bash
-go build -o bin/arborist
+go get golang.org/x/tools/cmd/goyacc
```

Comment:
could we put this step into the makefile?

Reply:
sure!